core/model/src/main/java/org/eclipse/sw360/antenna/model/license/LicenseInformation.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Bosch.IO GmbH 2020.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.model.license;
+
+import java.util.List;
+
+public abstract class LicenseInformation {
```

Comment:
Why didn't you use interface here?

Reply:
Because i referenced this class in the bindigs.xjb file and the licenseInformation type in the config.xsd is an abstract complexType. 
I will change this abstract class to an interface and introduce an abstract class in the `FromXmlLicenseInformationBuilder` and reference this in the bindings.xjb.